### PR TITLE
Fully disabling `console` usage for now at least

### PR DESCRIPTION
**Pull request by pshomov**
## What

Disable logging using `console` object

## Why

We want to get everyone to use our standard logging library

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against master before asking for a review

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     ],
     "no-arg": true,
     "no-bitwise": true,
-    "no-console": [true, "debug", "info", "time", "timeEnd", "trace"],
+    "no-console": true,
```
Comment (eirikurn) [inline tslint.json:30]: We're not using tslint. I think this file can be removed. Configure .eslintrc instead.

**Review by eirikurn — CHANGES_REQUESTED** [review]

File: tslint.json (hunk 1 above)
Comment (pshomov) [reply tslint.json:30]: 👍🏻

**Review by arnifreyrs — DISMISSED** [review]

**Comment by eirikurn** [comment]
@pshomov I see that .eslintrc is not an implicit dependency for all projects, so the CI did not lint all the projects. Did you lint the whole repo and are there no new errors? (Is that what almost destroyed your computer?)

I'd rather not break master with this.

**Comment by pshomov** [comment]
> @pshomov I see that .eslintrc is not an implicit dependency for all projects, so the CI did not lint all the projects. Did you lint the whole repo and are there no new errors? (Is that what almost destroyed your computer?)
> 
> I'd rather not break master with this.

I mean if it did not break the CI then it will not break master if that's the breakage you are referring to.

**Comment by pshomov** [comment]
> @pshomov I see that .eslintrc is not an implicit dependency for all projects, so the CI did not lint all the projects. Did you lint the whole repo and are there no new errors? (Is that what almost destroyed your computer?)
> 
> I'd rather not break master with this.

@eirikurn Just noticed `.eslint.react.json` which seems to be used by front-end apps. Do we want the `no-console` rule there as well? Isn't Sentry the front-end logging platform?

**Comment by eirikurn** [comment]
@pshomov no worries.  .eslint.react.json extends .eslintrc.

**Comment by pshomov** [comment]
> @pshomov no worries. .eslint.react.json extends .eslintrc.

I am aware, my question was if we would like to allow some console usage in the frontend.

**Comment by eirikurn** [comment]
Ah. Then no. We don't want any console.log in production by default.

**Review by eirikurn — APPROVED** [review]
Approved pending full-repo lint.

**Comment by oddsson** [comment]
Can we put a automerge label on this so it get's merged?

**Pull request closed without merging**